scripts/qa_crowbarsetup.sh

```diff
@@ -2437,6 +2482,11 @@ function deploy_single_proposal()
 
     # create proposal
     case "$proposal" in
+        nfs_client)
+            if [[ -n "$clusternodesservices" ]]; then
```

Comment:
Actually, that's the current style we have everywhere else, so I'd prefer to keep the same style.